lib/wallaby/dsl/finders.ex

```diff
@@ -13,6 +15,17 @@ defmodule Wallaby.DSL.Finders do
     do_find_elements(session, to_params(query))
   end
 
+  def wait_until_visible(session, query, opts \\ [])
+  def wait_until_visible(session, query, [{:count, count} | _] = _opts) do
+    retry fn ->
+      case all(session, query) do
+        elements when length(elements) == count -> elements
```

Comment:
It'd also be nice to wait for all without specifying a count. Maybe create a separate `wait_until_all_visible` or something? or have `wait_until_visible(session, ".main", count: :all)`. What do you think?